CHANGELOG.md

```diff
@@ -97,6 +145,23 @@ for downloading the blocks again.
 - Config file `wallet.conf` won't be read during test runs that was
   causing issues with some testing environments.
 
+### Chain changes
+
+- The method `getSpentView` accepts a `TXMeta` instead of `TX`.
+- The transaction index methods are now implemented at `node.txindex`:
+  - `getMeta(hash)`
+  - `getTX(hash)`
+  - `hasTX(hash)`
+  - `getSpentView(tx)`
+- The address index method is now implemented at `node.addrindex`:
+  - `getHashesByAddress(addrs)`
```

Comment:
The API changed to accept only `Address` instances, rather than `Address|String`

Reply:
Same for all `Address` related methods.

Reply:
Actually, this could still work if `Address.getHash` accepts strings. The jsdoc is outdated as it mentions `String|Address|Hash`, but a string returns:

`Error: Object is not an address.`

Possibly string handling was removed during `string - buffer` replacement.

Reply:
@tuxcanfly see: https://github.com/bcoin-org/bcoin/pull/773 and the other issues/PRS linked in that conversation.

Reply:
Oh, yes. And there are additional arguments now too.

Reply:
I think it's better to deprecate `String` addrs, although maybe we could tackle this later. Maybe just convert now like you're doing in that PR and fix the jsdoc.